app/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Providers\RouteServiceProvider;
+use Illuminate\Http\Request;
+use Illuminate\Foundation\Auth\AuthenticatesUsers;
+use Illuminate\Support\Facades\Auth;
+
+class LoginController extends Controller
+{
+    use AuthenticatesUsers;
+    /*
+    |--------------------------------------------------------------------------
+    | Login Controller
+    |--------------------------------------------------------------------------
+    |
+    | This controller handles authenticating users for the application and
+    | redirecting them to your home screen. The controller uses a trait
+    | to conveniently provide its functionality to your applications.
+    |
+* @param  \Illuminate\Http\Request $request
+    *
+    * @return Response
+    */
+
+
+    public function login(Request $request)
+    {
+        $user = $request->validate([
+            'email' => ['required', 'email'],
+            'password' => ['required'],
+        ]);
+
```

Comment:
if ($this->attemptLogin($request)) {
            return $this->redirectTo();
}